### PR TITLE
Merge Chrome and Firefox repositories, Search box fix

### PR DESCRIPTION
**Pull request by sarangjo**
- Since the source code is identical, I think it would be nice to have a common repository for both, with the README having instructions for installation on either platform!
- Facebook recently added a "Search in Messages" option at the top of the search bar on the left, so the search + enter functionality was not working. This should be fixed with this pull request.

### CODE REVIEW

File: src/messenger-shortcuts.js [hunk 1]
```diff
@@ -129,7 +129,8 @@ function focusMessageInput() {
 }
 
 function selectFirstSearchResult() {
-  var first = document.querySelector('ul[role="listbox"] li div');
+  var contacts = document.querySelectorAll('ul[role="listbox"]')[1];
```
Comment (guoguo12) [inline src/messenger-shortcuts.js]: When you only type a single letter into the search box, the "Search Messages" section doesn't show up, so the contacts `listbox` is the first section.

File: src/messenger-shortcuts.js [hunk 2]
```diff
@@ -148,7 +149,8 @@ function toggleInfo() {
 }
 
 function getSearchBar() {
-  return getByAttr('input', 'type', 'text');
+  return document.querySelector('input[type="text"][placeholder="Search Messenger"]');
```
Comment (guoguo12) [inline src/messenger-shortcuts.js]: Any reason for the extra condition? Are there cases where it doesn't work without it? (I'm worried that they'll quietly change the "Search Messenger" text at some point.)

File: src/messenger-shortcuts.js [hunk 3]
```diff
@@ -148,7 +149,8 @@ function toggleInfo() {
...
+  // return getByAttr('input', 'type', 'text');
```
Comment (guoguo12) [inline src/messenger-shortcuts.js]: You can safely delete this line. It's all in version control anyway. 😉

**Comment by guoguo12** [comment]
Thanks for the PR! See the comments above.

File: src/messenger-shortcuts.js (hunk 1 above)
Comment (sarangjo) [inline src/messenger-shortcuts.js]: Ooh that's a good catch, I'll add a condition based on the number of characters in the search box.

File: src/messenger-shortcuts.js (hunk 3 above)
Comment (sarangjo) [inline src/messenger-shortcuts.js]: Good point, I always keep old code out of habit 😜

File: src/messenger-shortcuts.js (hunk 2 above)
Comment (sarangjo) [inline src/messenger-shortcuts.js]: Oops this was an old code when I was trying to figure out a solution, I'll take that out.

File: src/messenger-shortcuts.js [hunk 4]
```diff
@@ -129,7 +129,13 @@ function focusMessageInput() {
 }
 
 function selectFirstSearchResult() {
-  var contacts = document.querySelectorAll('ul[role="listbox"]')[1];
+  var listboxes = document.querySelectorAll('ul[role="listbox"]');
+  // Checking if only a single character has been pressed
+  var contacts = listboxes[(getSearchBar().value.length() <= 1) ? 0 : 1];
+    contacts = document.querySelector('ul[role="listbox"]');
```
Comment (guoguo12) [inline src/messenger-shortcuts.js]: Did you convert your if-else block into a ternary and then forget to delete your old code?

Comment (sarangjo) [inline src/messenger-shortcuts.js]: Wth, sorry this is such shoddy code, my apologies! Good lord.

**Comment by guoguo12** [comment]
Thanks! Will publish to Chrome Web Store this weekend.

**Pull request merged**